gluten-core/src/main/scala/io/glutenproject/expression/BinaryExpressionTransformer.scala

```diff
@@ -55,6 +59,51 @@ class BinaryExpressionTransformer(
   }
 }
 
+
+class FromUnixTimeTransformer(sec: ExpressionTransformer, format: ExpressionTransformer,
+  timeZoneId: Option[String] = None, original: FromUnixTime)
+  extends ExpressionTransformer
+  with Logging {
+
+  override def doTransform(args: java.lang.Object): ExpressionNode = {
+    if (BackendsApiManager.getBackendName.equalsIgnoreCase(GlutenConfig.GLUTEN_VELOX_BACKEND)) {
+      throw new UnsupportedOperationException(s"not supported yet.")
+    }
+
+    val secNode = sec.doTransform(args)
+    var formatNode = format.doTransform(args)
+
+    // Only when format = 'yyyy-MM-dd HH:mm:ss' can we transfrom the expr to substrait.
+    if (!secNode.isInstanceOf[ExpressionNode] ||
+      !formatNode.isInstanceOf[StringLiteralNode] ||
+      formatNode.asInstanceOf[StringLiteralNode].getValue != "yyyy-MM-dd HH:mm:ss") {
+      throw new UnsupportedOperationException(s"not supported yet.")
+    }
+    formatNode = ExpressionBuilder.makeStringLiteral("%Y-%m-%d %R:%S")
+
+    val dataTypes = if (timeZoneId != None) {
+      Seq(original.sec.dataType, original.format.dataType, StringType)
+    } else {
+      Seq(original.sec.dataType, original.format.dataType)
+    }
+    val functionMap = args.asInstanceOf[java.util.HashMap[String, java.lang.Long]]
+    val functionId = ExpressionBuilder.newScalarFunction(functionMap,
+      ConverterUtils.makeFuncName(ExpressionMappings.FROM_UNIXTIME, dataTypes))
```

Comment:
use `substraitExprName: String` to pass substrait expression name, dont use `ExpressionMappings.FROM_UNIXTIME`, please refer to `HashExpressionTransformer`